tcms/static/js/testcase_actions.js

```diff
@@ -913,44 +913,6 @@ function removePlanFromCase(container, plan_id, case_id) {
   jQ('#plan_count').text(jQ('table#testplans_table').attr('count'));
 }
 
-function renderTagForm(container, parameters, form_observe) {
```

Comment:
Also remove `updateCaseTag` from this file.

Reply:
Done